app/templates/application.hbs

```diff
@@ -2,13 +2,22 @@
 
 <AuApp>
   <AppHeader />
-  <NavigationBar />
-
+  {{#if this.isIndex}}
+  <AuContentHeader
+    @titlePartOne="We geven je alle beslissingen"
+    @titlePartTwo="uit JOUW gemeente"
+  >
+    <img
+      sizes="50vw"
+      src={{"/assets/loket-header-1600.jpg"}}
+      alt="Foto van een laptop met daarop het vlaanderen logo."
```

Comment:
```suggestion
      alt=""
      {{!template-lint-disable require-valid-alt-text}}
```

Reply:
You're right but the linter won't allow a empty alt tag so can't commit this change :-)

Reply:
Yea, but the disable comment in the snippet should fix that?